addons/medical/functions/fnc_deserializeState.sqf

```diff
@@ -35,6 +35,22 @@ if !(_unit getVariable [QGVAR(initialized), false]) exitWith {
 
 private _state = [_json] call CBA_fnc_parseJSON;
 
+// Migration from old array wounding storage serialized in old versions (<= 3.15.0)
```

Comment:
Before merge this comment will need updating to whatever latest state is. Because I'm not currently involved in our release management strategy I think someone else who is should make the call on merging this.